Contribute/dotnet/dotnet-contribute.md

```diff
@@ -24,36 +24,32 @@ Issues for all these repositories are tracked at the [dotnet/docs](https://githu
+We appreciate community contributions to docs. The following list shows some guiding rules to keep in mind when you're contributing to the .NET documentation:
 
 - **DON'T** surprise us with large pull requests. Instead, file an issue and start a discussion so we can agree on a direction before you invest a large amount of time.
 - **DO** follow these instructions and [voice and tone](dotnet-voice-tone.md) guidelines.
 - **DO** use the [template](dotnet-style-guide.md) file as the starting point of your work.
 - **DO** create a separate branch on your fork before working on the articles.
-- **DO** follow the [GitHub Flow workflow](https://guides.github.com/introduction/flow/).
-- **DO** blog and tweet (or whatever) about your contributions, frequently!
+- **DO** follow the [GitHub flow](https://guides.github.com/introduction/flow/).
+- **DO** blog and tweet (or whatever) about your contributions if you like!
 
 Following these guidelines will ensure a better experience for you and for us.
 
 ## Make a contribution to .NET docs
 
-**Step 1:** Skip this step for small changes. If you're interested in writing new content or in thoroughly revising existing content, open an [issue](https://github.com/dotnet/docs/issues) describing what you want to do.
-
-The content inside the **docs** folder is organized into sections that are reflected in the Table of Contents (TOC). Define where the topic will be located in the TOC. Get feedback on your proposal.
+**Step 1:** If you're interested in writing new content or in thoroughly revising existing content, open an [issue](https://github.com/dotnet/docs/issues) describing what you want to do. The content inside the **docs** folder is organized into sections that are reflected in the Table of Contents (TOC). Define where the topic will be located in the TOC. Get feedback on your proposal.
 
 -or-
 
-You can also choose from existing issues for which community contributions are welcome. [Projects for .NET Community contributors](https://github.com/dotnet/docs/projects/35) lists many of the issues that are available for community contributors. Depending on your interests and level of commitment, you can choose from issues in the following categories:
+Choose an existing issue and address it. You can look at our [open issues](https://github.com/dotnet/docs/issues) list and volunteer to work on the ones you're interested in. We use the [up-for-grabs](https://github.com/dotnet/docs/labels/up-for-grabs) label to tag issues identified for community contribution. These usually require minimal context and are well suited for first issues. We encourage experienced contributors in our community to look at any issues of interest. When you find one, add a comment to ask if it's open. Depending on your interests and level of commitment, you can choose from issues in the following categories:
```

Comment:
Since we first wrote this, we've added the [good first issue](https://github.com/dotnet/docs/labels/good-first-issue) label. We should metnion that here.